lib/build-utilities/migrate-initializers.js

```diff
@@ -89,10 +89,8 @@ function _moveFastBootInitializers(project, rootPath) {
  * @param {Object} project
  */
 function _migrateAddonInitializers(project) {
-  const nodeModulesPath = path.join(project.nodeModulesPath);
-
-  project.addons.forEach((addon) => {
-    const currentAddonPath = path.join(nodeModulesPath, addon.name);
+  project.addons.forEach(addon => {
+    const currentAddonPath = path.join(addon.root);
```

Comment:
I think we can make this:

```
const currentAddonPath = addon.root;
```

Specifically, it seems odd to call path.join with only one arg...

Reply:
@rwjblue agree, i just pushed a simplified version.